_extensions/apiref.py

```
@@ -0,0 +1,64 @@
+    match = value_re.match(value)
+    if match is None:
+        return None, value
+    return match.group(1), match.group(2)
+
+
+def encode_doxygen(value):
+    value = value.split("/")[-1]
+    try:
+        return "".join(DOXYGEN_LOOKUP[s] for s in value)
+    except KeyError:
+        raise ValueError("Unknown character in doxygen string! '{}'".format(value))
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)